docs/usage/serve.rst

```diff
@@ -3,14 +3,16 @@
 Examples
 ~~~~~~~~
 
-borg serve has special support for ssh forced commands (see ``authorized_keys``
-example below): it will detect that you use such a forced command and extract
-the value of the ``--restrict-to-path`` option(s).
+``borg serve`` has special support for ssh forced commands (see ``authorized_keys``
+example below): if the environment variable SSH_ORIGINAL_COMMAND is set it will
+ignore some options given on the command line and use the values from the
+variable instead. This only applies to a carefully controlled allowlist of safe
+options. This list currently contains:
 
-It will then parse the original command that came from the client, makes sure
-that it is also ``borg serve`` and enforce path restriction(s) as given by the
-forced command. That way, other options given by the client (like ``--info`` or
-``--umask``) are preserved (and are not fixed by the forced command).
+- Options that control the amount of debug information printed
```

Comment:
Options that control the log level and debug topics printed such as --verbose, --info, --debug, --debug-topic, etc.